node/generate-with-together-ai/src/main.js

```diff
@@ -0,0 +1,113 @@
+import { Client, ID, InputFile, Storage } from 'node-appwrite';
+import { getStaticFile, throwIfMissing } from './utils.js';
+import { fetch } from 'undici'
+
+export default async ({ req, res, error }) => {
+  throwIfMissing(process.env, [
+    'TOGETHER_API_KEY',
+    'APPWRITE_API_KEY',
+    'APPWRITE_FUNCTION_PROJECT_ID',
+    'APPWRITE_BUCKET_ID'
+  ]);
+
+  if (req.method === 'GET') {
+    return res.send(getStaticFile('index.html'), 200, {
+      'Content-Type': 'text/html; charset=utf-8',
+    });
+  }
+
+  const models = {
+    'text': 'mistralai/Mixtral-8x7B-Instruct-v0.1',
+    'image': 'stabilityai/stable-diffusion-xl-base-1.0'
```

Comment:
does the API only support text and images? The README.md says audio as well on Line 3 and 21